content/millennium/r4/base/individuals/patient.md

```diff
@@ -80,6 +81,9 @@ Notes:
   * May be provided twice using the prefixes `le` and `ge` to indicate a date range
   * May be provided once using one of the following prefixes to imply a date range: `ge`, `le`, `gt`, `lt`, `eq`
   * Must not be provided with a time component
+* The `_revinclude` parameter may be provided once with the value `Provenance:target`. Example: `_revinclude=Provenance:target`
+* The `_revinclude` parameter may be provided with the `_id/identifier` parameter. Example: `_id=629928,identifier=urn:oid:1.1.1.1.1.1|10000360&_revinclude=Provenance:target`
```

Comment:
I am confused,
we give _revinclude with only _id/identifier ?

Reply:
no we support with patient also , but since this is patient resource only we cant search with patient parameter